tests/test_env_setup.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-export TEST_CODECHECKER_PACKAGE_DIR="$(dirname "$BASH_SOURCE")/codechecker_package"
+export TEST_TESTS_DIR=$(readlink -f "$(dirname "$BASH_SOURCE")")
+
+export TEST_CODECHECKER_PACKAGE_DIR="$TEST_TESTS_DIR/tmp"
```

Comment:
I would prefer to build the package outside the repository directory tree.
'mktemp -d' could be used in bash to create a temporary directory.